### PR TITLE
Add new extension Australian Veteran Status 

### PR DESCRIPTION
**Pull request by mjosborne1**
- New extension to patient for Australian Veteran Status https://github.com/hl7au/au-fhir-base/issues/790
- Updated AU Base Patient, changes.md, patient-example-1, terminology.md

### CODE REVIEW

**Comment by mjosborne1** [comment]
release date for the Australian Veteran Status Codesystem and Valueset into NCTS is Weds 27th Sept 2023

File: input/pagecontent/changes.md
Comment (dt-r) [inline input/pagecontent/changes.md:1]: Nice changes in this file, just missing ":" at the end of "Changed Patient.extension"

File: input/pagecontent/terminology.md
Comment (dt-r) [inline input/pagecontent/terminology.md:1]: I note an additional change in this file on top of veteran status. It's ok, it improves the alphabetical order.

File: input/resources/structuredefinition-veteran-status.xml
Comment (dt-r) [inline input/resources/structuredefinition-veteran-status.xml:1]: Need to change `StructureDefinition.status` to `draft` to match FMM 0

**Review by dt-r — CHANGES_REQUESTED** [review]
Hi Michael,

Thanks for your PR. Just need to make a change to StructureDefinition, and add punctuation in changes.md

**Comment by mjosborne1** [comment]
I have made changes as requested and committed to au-veteran-status branch.

**Review by dt-r — APPROVED** [review]

**Pull request merged**